vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/add.html

```diff
@@ -0,0 +1,95 @@
+<html layout:decorate="~{layouts/main_staff}">
+<head>
+<link th:href="@{/resources/multiselect/css/multiselect.css}"
+    type="text/css" rel="stylesheet">
+<script th:src="@{/resources/multiselect/js/multiselect.min.js}"></script>
+
+<script>
+$(document).ready(function() {
```

Comment:
have you made sure that the script is loaded into the final page?

Reply:
Yes, the script is getting loaded and functionality of adding slides and removing slides by double clicking on the slide name is also working fine.